packages/orbit-components/src/utils/mediaQuery/index.ts

```diff
@@ -1,36 +1,31 @@
-import { css } from "styled-components";
-
 import type { Theme } from "../../defaultTheme";
-import { QUERIES } from "./consts";
-import type { MediaQueries } from "./types";
+
+export enum QUERIES {
+  MEDIUMMOBILE = "mediumMobile",
+  LARGEMOBILE = "largeMobile",
+  TABLET = "tablet",
+  DESKTOP = "desktop",
+  LARGEDESKTOP = "largeDesktop",
+}
 
 export const TOKEN = {
   mediumMobile: "widthBreakpointMediumMobile",
   largeMobile: "widthBreakpointLargeMobile",
   tablet: "widthBreakpointTablet",
   desktop: "widthBreakpointDesktop",
   largeDesktop: "widthBreakpointLargeDesktop",
-};
+} as const;
+
+export interface GetBreakpointWidth {
+  (name: keyof typeof TOKEN, theme: Theme): string;
+  (name: keyof typeof TOKEN, theme: Theme, pure: false): string;
+  (name: keyof typeof TOKEN, theme: Theme, pure: true): number;
+}
 
-export const getBreakpointWidth = (
-  name: keyof typeof TOKEN,
+export const getBreakpointWidth: GetBreakpointWidth = (
+  name: string,
```

Comment:
Why are we relaxing the requirement here?

Reply:
It is an interesting thing, actually. So, according to the interface defined above, the return type of this function is `string | number` but if I defined this prop to be `keyof typeof TOKEN` it will consider `theme.orbit[TOKEN[name]]` even though **it will** return a number, because `TOKEN[name]` **always** refers to a key that corresponds to a number in `theme.orbit`.

However, relaxing the requirement here does not seem to be problematic, as I tested passing a random string to this function on [this file](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/hooks/useMediaQuery/index.ts#L54) and Typescript complained, saying that it was not one of the breakpoints. So I would say we are safe here… WDYT?

Reply:
Do you even need to type the arguments if you're typing the function already? 🤔

Reply:
I guess 😞 that was my first thought was well!

<img width="1033" alt="Screenshot 2024-03-28 at 17 09 25" src="https://github.com/kiwicom/orbit/assets/6265045/8319625e-b27d-459b-ba0b-933b2881a19d">

Reply:
So this didn't work out in the end?

Reply:
Thanks for raising. Solved in #4327